packages/terra-date-picker/src/react-datepicker/time.jsx

```diff
@@ -15,14 +15,41 @@ import {
 
 export default class Time extends React.Component {
   static propTypes = {
+    /**
+     * Format of the selected time.
+     */
     format: PropTypes.string,
+    /**
+     * Interval between 2 consecutive times on the time picker.
+     */
     intervals: PropTypes.number,
+    /**
+     * Selected value of time.
+     */
     selected: PropTypes.object,
+    /**
+     * A callback function to execute when time is picked.
+     */
     onChange: PropTypes.func,
+    /**
+     * A button to select current time.
```

Comment:
Does this specify the name of the button? Like here https://github.com/cerner/terra-framework/pull/785/files#diff-f0779fb2b55a5a8b2bef87c374f19eceR219